dom.bs

```diff
@@ -8724,6 +8724,20 @@ associated <a for="/">element</a>'s
   <p>Throws a <code>TypeError</code> exception if the associated attribute has no supported tokens defined.
 </dl>
 
+<p>The <a method for=DOMTokenList lt="supports()">supports()</a> method can be used to detect support for {{DOMTokenList}} tokens:
+
+<pre class=lang-javascript>
+function domTokenListSupports(tokenList, token) {
+    if (!tokenList || !tokenList.supports) {
+        return;
+    }
+    return tokenList.supports(token);
+};
```

Comment:
@annevk's specs don't use semicolons. Need to fix lines 8732, 8734, 8735, 8737, 8738.